ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -110,7 +110,7 @@ private[apiserver] final class ApiSubmissionService private (
 
   override def submit(request: SubmitRequest): Future[Unit] =
     withEnrichedLoggingContext(logging.commands(request.commands)) { implicit loggingContext =>
-      logger.debug("Submitting transaction")
+      logger.info("Submitting transaction")
```

Comment:
@kamil-da , don't you feel logging every submission at info level might turn out to be a bit verbose?

Reply:
@oliverse-da Thank you for the comment.
Actually I think it may be more reasonable to lower the level to debug.
This change was explicitly requested by @mziolekda , so his input might be valuable here as well.